docset/winserver2025-ps/smbshare/New-SmbGlobalMapping.md

```diff
@@ -58,8 +70,30 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -BlockNTLM
+
+Specifies whether to block NT LAN Manager (NTLM) authentication. If this parameter is set to
```

Comment:
differs from previous text used in new-smbshare PR

Reply:
New-SmbMapping PR